homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -392,7 +395,7 @@ async def async_clean_segment(self, segments):
             segments=segments,
         )
 
-    async def async_clean_zone(self, zone, repeats=1):
+    async def async_clean_zone(self, zone, repeats=1) -> None:
```

Comment:
```suggestion
    async def async_clean_zone(self, zone: list[Any], repeats: int=1) -> None:
```
I'm not sure what's the exact contents for the zones list.